src/ui/ui.cpp

```diff
@@ -46,4 +46,75 @@ void UI::slip_tracks(int track_id, int offset) {
 
     slip -> slip_track(track_id, offset);
     std::cout << "Slipped track " << track_id << " by " << offset << " samples." << std::endl;
-}
\ No newline at end of file
+}
+
+void UI::save_loop(int loop_num) {
+    std::cout << "Loop loaded or created" << std::endl;
+    db_manager.create_loop(std::to_string(loop_num));
+    int id = db_manager.find_loop_id(std::to_string(loop_num));
+
+    std::cout << "From HERE" << std::endl;

```

Comment:
we can rewrite these two cout statements into one line that would make it easier to read in the terminal. 

cout << "some text: " << some_value << std::endl;

Reply:
I removed these lines since they were just helping me debug. but provided no real information